new-lamassu-admin/src/components/Status.js

```diff
@@ -0,0 +1,89 @@
+import React, { memo } from 'react'
+
+import Chip from '@material-ui/core/Chip'
+
+import { withStyles } from '@material-ui/core/styles'
+import { tomato, mistyRose, secondaryColorDarker as spring4, inputFontWeight, spring3, smallestFontSize, inputFontFamily } from '../styling/variables'
+
+const green = theme => ({
+  root: {
+    backgroundColor: spring3,
+    borderRadius: 4,
+    margin: theme.spacing(0.5, 0.25),
+    height: 18
+  },
+  label: {
+    fontSize: smallestFontSize,
+    color: spring4,
+    fontWeight: inputFontWeight,
+    fontFamily: inputFontFamily,
+    paddingRight: 4,
+    paddingLeft: 4
+  }
+})
+
+const orange = theme => ({
+  root: {
+    backgroundColor: mistyRose,
+    borderRadius: 4,
+    margin: theme.spacing(0.5, 0.25),
+    height: 18
+  },
+  label: {
+    fontSize: smallestFontSize,
+    color: '#ff7311',
```

Comment:
Don't we have a named color for this? We should.

We might not need the three themed objects, I'm thinking we can have one and a class to change label color.